lib/tasks/drafts.rake

```diff
@@ -21,6 +21,24 @@ namespace :drafts do
     end
   end
 
+  desc 'Load full draft into database'
+  task load_full_variable: :environment do
```

Comment:
`load_full_variable` is already in this file

Reply:
Hm. Look at that.

Reply:
Fixed that up -- good catch (the one i added was broke anyway 🤦‍♂️ )